src/main/java/io/cdap/e2e/utils/SeleniumHelper.java

```diff
@@ -141,4 +147,18 @@ public static boolean verifyElementPresent(String locator) {
       return false;
     }
   }
+
+  public static String readErrors(String property) throws IOException {
+    Properties prop = new Properties();
+    InputStream input = null;
+    try {
+      input = SeleniumHelper.class.getResourceAsStream("/ErrorMessage.properties");
+      prop.load(input);
+      return prop.getProperty(property);
```

Comment:
This is very inefficient. We should not load it on every call

Reply:
made it a static load